fixtures/gists-app/app/routes/__layout.tsx

```diff
@@ -1,10 +1,22 @@
+import type { LoaderFunction } from "remix";
+import { useLoaderData } from "remix";
 import { Outlet } from "remix";
 
+export interface RouteData {
+  nested: boolean;
+}
+
+export let loader: LoaderFunction = () => {
+  let data: RouteData = { nested: true };
+  return data;
+};
+
 export default function LayoutTest() {
+  let data = useLoaderData<RouteData>();
   return (
     <div data-test-id="_layout">
       <h1>Layout Test</h1>
-      <Outlet />
+      <Outlet context={data} />
```

Comment:
We don't want people using the `context` prop to share loader data because they can already access that more easily via `useMatches`. Do you mind updating this test to use something else so people reading the tests don't get the wrong idea? I'm writing more docs on this too.

```suggestion
      <Outlet context={contextData} />
```